scripts/clean_all.sh

```diff
@@ -10,25 +10,20 @@ cd ${RULES_NODEJS_DIR}
 
 echo_and_run() { echo "+ $@" ; "$@" ; }
 
-echo_and_run rm -rf ./release
 echo_and_run rm -rf `find . -type d -name node_modules -prune`
-
 echo_and_run bazel clean --expunge
 
-${RULES_NODEJS_DIR}/scripts/clean_e2e_all.sh
-${RULES_NODEJS_DIR}/scripts/clean_examples_all.sh
-
-(
-  cd internal/e2e
-  for subDir in $(ls) ; do
-    [[ -d "${subDir}" ]] || continue
-    (
-      cd ${subDir}
-      if [[ -e 'WORKSPACE' ]] ; then
-        printf "\n\nCleaning /internal/e2e/${subDir}\n"
+readonly workspaceRoots=("e2e" "examples" "packages")
+for workspaceRoot in ${workspaceRoots[@]} ; do
+  (
+    readonly workspaceFiles=($(find ./${workspaceRoot} -type f -name WORKSPACE -prune))
+    for workspaceFile in ${workspaceFiles[@]} ; do
+      (
+        readonly workspaceDir=$(dirname ${workspaceFile})
+        printf "\n\nCleaning ${workspaceDir}\n"
+        cd ${workspaceDir}
         echo_and_run bazel clean --expunge
```

Comment:
do you think we actually need this?

Reply:
Changed to clean_nested_workspaces.sh. Useful if you cd into a nested workspaces and run a it manually and want to clean it up after from root. If there is a `dist` folder left over in the nested workspace it messes up the root workspace's build & test.